indexing-service/src/main/java/org/apache/druid/indexing/worker/TaskAnnouncement.java

```diff
@@ -35,6 +35,8 @@
  */
 public class TaskAnnouncement
 {
+  public static final String TASK_ID_KEY = "id";
```

Comment:
In my view it's closer to being in the 'model' aka in the `TaskAnnouncement` record, since we want the `taskId`. It should also act as a prompt so that if the `TaskAnnouncement field is renamed at any point then compiler will flag an issue.

Reply:
maybe we can make this more clear by including a `static String getTaskIdKeyName()` { return "id"; }`, which can be called from `ZkWorker`, thoughts?

Reply:
Oh, I see the confusion; I created it here but didn't land it in the PR. Will include in fix.